c2corg_ui/static/js/documentdetailsview.js

```diff
@@ -102,5 +136,191 @@ app.ViewDetailsController.prototype.openTab = function(tab) {
       $('.tab.' + tab).show();
     }
   }
+};
+
+
+/**
+ * copied and adapted from http://codepen.io/jeffpannone/pen/GpKOed
```

Comment:
I digged on this issue and it appears public pen stuff is supposed to be MIT.
So it should be fine for us to use:
- https://blog.codepen.io/legal/licensing/;
- http://www.gnu.org/licenses/license-list.en.html#X11License.